components/FundStatus/FundStatus.tsx

```diff
@@ -48,6 +49,30 @@ const FundStatus = ({ fin, ver, amm }: FundStatusProps) => {
             <dt className="ml-0.5">{useMessage(`fund.status.${s.key}`)}</dt>
           </div>
         ))}
+        <Tooltip
+          content={
+            <aside className="uppercase">
+              <dt className="mb-1">
+                <span className="text-green-500">
```

Comment:
`dt`s (definition term) need to have a corresponding `dd` (definition description) as a sibling.

You could loop through `['fin', 'ver', 'amm']` and have somethign like:
```tsx
<dl>
{['fin', 'ver', 'amm'].map((x) => (
    <FormattedMessage
      id="fund.status.tooltip"
      values={{
        abbr: <dt className="mb-1"><span className="text-green-500">useMessage(`fund.status.${x}`)</span></dt>,
        description: <dd>{useMessage(`fund.status.tooltip.${x}.text`)}</dd>
      }}
    />
  </dt>
)}
</dl>
```
And the message would be:
```js
"fund.status.tooltip": "{abbr} = {description}"

```